### ISSUE TITLE
AAACT Assessment (WCAG)

### ISSUE THREAD

**Issue by GuillaumeTurcotte** [dto-btn/ssc-assistant#198]
- [ ] Initial assessment of WCAG lvls

**Comment by harshakakumanu** [comment]
Initial assessment as follows 
![Screenshot 2024-09-10 103955](https://github.com/user-attachments/assets/d2974fe5-81b1-4538-a453-7119db3baa29)

**Comment by harshakakumanu** [comment]
- Add Missing form label

**Comment by harshakakumanu** [comment]
One form label cannot be resolved because its a false positve as its aria-hidden = "true"

https://github.com/mui/material-ui/issues/35580

**Comment by chrisplate** [comment]
MDN (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden) says:

aria-hidden="true" should not be added when:

The HTML [hidden] attribute is present
The element or the element's ancestor is hidden with [display: none]
The element or the element's ancestor is hidden with [visibility: hidden]

The textarea in question here has a style of visibility: hidden

So why is this a "false positive"?
